src/st/no3/48A84.c

```diff
@@ -236,7 +236,121 @@ void EntityUnkId30(Entity* self) {
     }
 }
 
-INCLUDE_ASM("st/no3/nonmatchings/48A84", EntityUnkId31);
+void EntityUnkId31(Entity* self) {
+    Entity* entity;
+    u16* hitboxPtr;
+    u16 animFrameIdx;
+    u16 animCurFrame;
+    s16 i;
+
+    entity = self - 2;
+    if (self->step == 0) {
+        InitializeEntity(D_80180B24);
+        self->zPriority++;
+        CreateEntityFromCurrentEntity(ITEM_NO_CAPE, self + 1);
```

Comment:
When creating entities, we should use the EntityIDs enum. Since it doesn't have an entry for 0x30, you can leave this as 0x30 for now. But ITEM_NO_CAPE should be reserved for situations where we're dealing with equippable items. If you're not familiar with enums I'm happy to elaborate.

Reply:
@bismurphy think I get you, I'm referencing stuff here in `items.h` which is probably incorrect, but `E_EXPLOSION` is defined in `no3.h` so that one is known correct?

And 0x30 / 0x32 are not in `no3.h` yet so I can leave these as hex values?

Reply:
Right. You can leave them as hex values, or if you want to be extra cool, you could make new entries in no3.h and call them E_ID_30, E_ID_31, E_ID_32. Your call, we aren't requiring using enums for function calls like this as of right now.

Reply:
Makes sense, I've added the stub entities as you mentioned, might as well save someone a tiny bit of future effort 😄